docs/topics/cache.txt

```diff
@@ -401,6 +403,21 @@ behavior. These arguments are provided as additional keys in the
     On some backends (``database`` in particular) this makes culling *much*
     faster at the expense of more cache misses.
 
+  Memcached backends pass the contents of :setting:`OPTIONS <CACHES-OPTIONS>`
+  as keyword arguments to the client constructors, allowing for more advanced
+  control of client behavior. For example usage, see below.
+
+  .. versionchanged:: 1.11
+
+      Memcached backends can now be configured using ``OPTIONS``.
```

Comment:
This is a little confusing because right after it you say how configuring pyblic used 'behaviors' in OPTIONS. Add something like "In older versions, you could pass pyblic behavior settings.." (Maybe combine it with the deprecation note)
Also move, these annotations to the bottom of the section.